### DISCUSSION TITLE
posts/2023/10/late-september-summary/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#23] [General]
# posts/2023/10/late-september-summary/

https://guozheng.rbind.io/posts/2023/10/late-september-summary/

**Comment by liechi** [comment]
>我忍不住打趣地和小杨师姐说总有曲终人散、人走茶凉的时刻

这里“人走茶凉”似乎用错了。不过，你也说了是“打趣”，不知是不是有我没理解到的调侃意味：）

你笔下的“兵哥”真挺好啊，在生命中的一段时期有这么个亦师亦友的人陪伴属实幸运。

**Reply by residualsun1** [reply]
> 这里“人走茶凉”似乎用错了。

确实错啦。我搜了一下“人走茶凉”，原来是比喻世态炎凉，人情淡漠。当时没想太多，望文生义地理解成“天下没有不散的宴席”的意思了哈哈。

> 你笔下的“兵哥”真挺好啊，在生命中的一段时期有这么个亦师亦友的人陪伴属实幸运。

是呀！我每次和兵哥待在一起的时候都会有种不真实的感觉。希望，同时也相信我们还会继续保持联系(●'◡'●).
